content/maxwell3_fdem/emi_tutorial/index.rst

```diff
@@ -0,0 +1,44 @@
+.. _emi_tuotorial_index:
+
+EM induction tutorial
+=====================
+
+The fundamental principles of electromagnetic (EM) induction can be understood with a simple geometry involving three elements each represented by a loop of conducting wire as shown in :numref:`Concepts_3loops`.
+
+- Loop1: is the transmitter. It carries a current that produces an associated magnetic field. This magnetic field is called the primary field. For induction experiments the current in the transmitter varies with time and hence the magnetic field varies with time.
```

Comment:
Here, Labels that are the same as in the top figure would be helpful (ie. `transmitter (Tx)`)